gulpfile.js

```diff
@@ -67,7 +67,7 @@ var autoRestExe;
 gulp.task('codegen', function(cb) {
     if (autoRestVersion.match(/[0-9]+\.[0-9]+\.[0-9]+.*/) ||
         autoRestVersion == 'latest') {
-            autoRestExe = 'autorest ---version=' + autoRestVersion;
+            autoRestExe = 'autorest-beta ---version=' + autoRestVersion;
```

Comment:
Need update package.json

Reply:
I think package.json is not needed. It just calls autorest-beta on exec. No dependency on nodejs.